content/docs/tasks/traffic-management/egress.md

```diff
@@ -155,23 +162,30 @@ to set a timeout rule on calls to the httpbin.org service.
 ## Calling external services directly
 
 If you want to completely bypass Istio for a specific IP range,
-you can configure the source service's Envoy sidecar to prevent it from
+you can configure the Envoy sidecars to prevent them from
 [intercepting](/docs/concepts/traffic-management/request-routing/#communication-between-services)
-the external requests. This can be done using the `--includeIPRanges` option of
-[istioctl kube-inject](/docs/reference/commands/istioctl/#istioctl kube-inject)
-when starting the service.
+the external requests. This can be done by setting the `global.proxy.includeIPRanges` variable of
+[Helm](/docs/setup/kubernetes/helm-install/#customization-with-helm) and updating the `ConfigMap` _istio-sidecar-injector_ by `kubectl apply`. After _istio-sidecar-injector_ is updated, the value of `global.proxy.includeIPRanges` will affect all the future deployments of the application pods.
 
-The simplest way to use the `--includeIPRanges` option is to pass it the IP range(s)
+The simplest way to use the `global.proxy.includeIPRanges` variable is to pass it the IP range(s)
 used for internal cluster services, thereby excluding external IPs from being redirected
 to the sidecar proxy.
 The values used for internal IP range(s), however, depends on where your cluster is running.
-For example, with Minikube the range is 10.0.0.1&#47;24, so you would start the sleep service like this:
+For example, with Minikube the range is 10.0.0.1&#47;24, so you would update your `ConfigMap` _istio-sidecar-injector_ like this:
 
-```command
-$ kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml --includeIPRanges=10.0.0.1/24)
+```plain
```

Comment:
This command does not work, here should be `helm template` but not `helm install`

Reply:
One more thing, the folder `template` was not packaged into release package, we need to package it as well, otherwise, we will not able to find the file of `templates/sidecar-injector-configmap.yaml`.

Reply:
@gyliu513 Template is packaged, I use https://github.com/istio/istio/releases/tag/0.8.0, it is there.

Reply:
@gyliu513 I have fixed `helm template`, good catch, thanks!

Reply:
Yes, the latest package include the template. Then the only comment is update to `helm template`